webapp-django/crashstats/crashstats/models.py

```diff
@@ -987,11 +987,19 @@ def get(self, bugs):
             url = settings.BZAPI_BASE_URL + (
                 '/bug?id=%(bugs)s&include_fields=%(fields)s' % params
             )
-            response = requests_retry_session().get(
+            response = requests_retry_session(
+                # BZAPI isn't super reliable, so be extra patient
+                retries=5,
+                # 502 = Bad Gateway
+                # 504 = Gateway Time-out
+                status_forcelist=(500, 502, 504),
```

Comment:
This utility function is a thin thing on top of `urllib3.util.retry.Retry` which is vendored into `requests`. 
https://github.com/shazow/urllib3/blob/master/urllib3/util/retry.py#L152

I'm just trying to map parameters to that. I see no `should_retry` thing natively in there.